src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsService.java

```diff
@@ -477,6 +481,19 @@ private boolean updateRequiredByDestination(FlowPatch flowPatch, Flow flow) {
     }
 
     private boolean updateRequiredByDiverseFlowIdField(FlowPatch flowPatch, Flow flow) {
+        Set<Integer> patchVlanStatistics = flowPatch.getVlanStatistics();
+        if (patchVlanStatistics == null) {
+            return false;
+        }
+
+        if (Objects.equals(patchVlanStatistics, flow.getVlanStatistics())) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private boolean updateRequiredByVlanStatistics(FlowPatch flowPatch, Flow flow) {
```

Comment:
I guess method names are wrong. Method above is about vlan stats but it has name `updateRequiredByDiverseFlowIdField`

Reply:
Fixed